app/src/main/java/pl/org/akai/movies/fragments/SearchMovieFragment.kt

```diff
@@ -29,6 +29,8 @@ class SearchMovieFragment : BaseFragment() {
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
 
+        movieAdapter = MovieAdapter(arrayListOf(), this.findNavController())
+
```

Comment:
Lepiej przekazać funkcję i nawigację zostawić we fragmencie przerobię to wieczorem